ql/config.ansi.hpp.in

```diff
@@ -0,0 +1,123 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
```

Comment:
I'd move this file into `${CMAKE_CURRENT_SOURCE_DIR}/cmake` to avoid confusion with `.in` files used by automake.